src/status_im/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -19,26 +17,21 @@
     :description                     (i18n/label :t/enable-notifications-sub-title)
     :description-accessibility-label :notifications-sub-title}])
 
-(defn- finish-onboarding
-  [notifications-enabled?]
-  (rf/dispatch [:push-notifications/switch notifications-enabled?])
-  (shell.utils/change-selected-stack-id shell.constants/default-selected-stack true nil)
```

Comment:
`change-selected-stack-id` is not needed anymore - https://github.com/status-im/status-mobile/pull/21634

Reply:
Marked the PR as draft, as it should be merged after: https://github.com/status-im/status-mobile/pull/21634

Reply:
ready for review.